### PR TITLE
About page reoriented to open source

### PR DESCRIPTION
**Pull request by lamagnifica**
## [DOC-753](https://openedx.atlassian.net/browse/DOC-753)

An OSPR made a change to the About page for open edX and edX Edge sites. That led to the discovery that our descriptions and examples for the About page and the learner dashboard used edx.org examples exclusively. These pages are styled differently than the OS and Edge versions, and they are not actually under the control of course teams.
### Date Needed

Initiating feature was released in July. 
### Reviewers
- [ ] Doc team review (copy edit): @catong or @srpearce 
- [ ] PM review: @mrrudnick

FYI @sarina 
### Testing
- [x] Ran ./run_tests.sh without warnings or errors
### HTML Version

The main file changed is the creating_new_course.rst, along with the scheduling_course.rst and setting_up_student_view.rst that follow it.
Here is the OS version of the guide:
http://draft-20151026-about-page.readthedocs.org/en/latest/building_course/creating_new_course.html
Here is the Partner version of the guide, which includes information about differences between working on a course on Edge vs. edx.org.
http://draft-20151026-about-page-partners.readthedocs.org/en/latest/building_course/creating_new_course.html
### Post-review
- [ ] Update change log
- [ ] Squash commits

### CODE REVIEW

File: en_us/shared/building_and_running_chapters/building_course/scheduling_course.rst [hunk 1]
```diff
@@ -17,36 +20,8 @@ Overview
 The start and end dates you set for your course and for course enrollment are
 important for prospective and current learners. As soon as enrollment starts,
 prospective learners can see your course in the course catalog, view the
-course About page, and enroll in the course. Current learners see your course
-start or end date on their dashboards.
-
-.. _View Start and End Dates:
-
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/scheduling_course.rst]: moved to end of file

**Comment by sarina** [comment]
@lamagnifica these changes only apply to the open source docs? The screen shots of edx.org still apply to edX courses

**Comment by lamagnifica** [comment]
@sarina my thinking was that since only PMs, and not course teams, alter this info for courses running on edX.org, the OS/Edge-looking ones are more generally applicable to all instead of the images from our Drupal site, and are also similar enough as to not be terribly misleading. Personally I'd rather reduce the number of images even further than I have, though. What do you think? Will people need an edx.org picture and an Edge picture if they're partners, but only an Edge picture if they're OS? or can we move further in the direction of relying on any reader to be using the doc concurrently with the software, rather than reading without having access to the software?

**Comment by sarina** [comment]
My worry is that people may think they can edit info on their edX.org
courses when the reality is they cannot.

**Comment by lamagnifica** [comment]
ah, ok. I will put the info (for partners only about edx.org only) all in one place, at the beginning of the two files for scheduling and setting up info, rather than buried throughout.

**Comment by sarina** [comment]
Is there someone on the PM side who would be good to look at the edx.org
documentation? I admit I don't understand that workflow anymore; all I know
about it is that course teams cna't edit the drupal site themselves, so
some edits they may make in Studio that they may *think *will affect the
drupal site will not.

**Comment by lamagnifica** [comment]
Hello @mhoeber @mrrudnick and @sarina , welcome back. I met with Mark R to discuss what partners can/cannot do when they're setting up course on Edge and edx.org, and how we want to position ways for them to get help. 
Because of a new automated process that stops extraneous spaces out of our rst files, I suggest using Split view to look at the files. I've tagged the most interesting bits. And, I also have HTML versions of both files available if it's easier to compare the different versions and see the images in place -- see the description above for links.

File: en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst [hunk 2]
```diff
@@ -1,134 +1,151 @@
...
-This topic describes how to create and set up your course with Studio.
+This topic describes how to use Studio to create and set up a course.
 
-.. contents:: Section Contents
+.. contents::
   :local:
   :depth: 1
 
-Another way to create a course is to re-run an existing course. See
-:ref:`Rerun a Course`.
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst]: the paragraph that follows was added for partners

File: en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst [hunk 3]
```diff
@@ -1,134 +1,151 @@
...
-   change the URL after the course is created, you must contact edX through the
-   Help site (http://help.edge.edx.org). Additionally, because this information
-   becomes part of your course URL, the total number of characters in the
-   following four fields must be 65 or fewer.
-   
-  .. image:: ../../../shared/building_and_running_chapters/Images/new_course_info.png
-     :width: 600
-     :alt: Image of the course creation page in Studio.
+.. _Identifying the Course:
+
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst]: I moved this section up, out of the creation step.

File: en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst [hunk 4]
```diff
@@ -1,134 +1,151 @@
...
+  Your Course`.
+
+The organization, course number, and course run values that you enter are used
+to create the URL for your course. Because URLs do not have an unlimited
+length, the total number of characters that you enter for these values must be
+65 or fewer.
+
+.. important:: The values that you enter for the organization, course number,
+ and course run cannot be edited after you create your course.
+
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst]: ^^ replaces advice to contact support per Mark R's input

File: en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst [hunk 5]
```diff
@@ -1,134 +1,151 @@
...
+To edit a course, select the course name. The Studio **Course Outline** page
 appears.
 
-.. _Use the Course Checklist:
+.. Topic commented out per M. Rudnick as the course checklist is sending people to info that is not just out of date, it's bad. - Alison 30 Oct 15
 
-************************
-Use the Course Checklist
-************************
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst]: I commented out this entire section per Mark R's advice. The Course Checklists aren't just out of date, they point people to "bad" info

File: en_us/shared/building_and_running_chapters/building_course/scheduling_course.rst [hunk 6]
```diff
@@ -17,55 +20,34 @@ Overview
...
+.. contents::
+  :local:
+  :depth: 1
 
 .. only:: Partners
 
-  .. note::
-    For courses on edX.org, you must communicate the course start and end dates
-    and times to your edX Partner Manager to ensure the information is accurate
-    on the course About page.
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/scheduling_course.rst]: I corrected this note: it's the enrollment dates that teams can't edit for edx.org, not start/end or all dates

File: en_us/shared/building_and_running_chapters/building_course/scheduling_course.rst [hunk 7]
```diff
@@ -131,11 +113,12 @@ the course. Ensure that the enrollment end date is late enough to allow
 learners adequate time to enroll. The enrollment end date cannot be later than
 the course end date.
 
-.. important:: 
-  For partner courses on edx.org, when the enrollment end date passes,
-  the course is no longer listed in the course catalog. EdX encourages you to
-  keep enrollment open as long as possible. For more information, contact your
-  edX Partner Manager.
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/scheduling_course.rst]: made the following note for Partners only. Removed advice to call your PM.

File: en_us/course_authors/source/getting_started/get_started.rst [hunk 8]
```diff
@@ -11,16 +11,16 @@ create your first course.
  :local:
  :depth: 1
 
-If you are using an instance of Open edX, some specifics in this section might 
+If you are using an instance of Open edX, some specifics in this section might
```
Comment (sarina) [inline en_us/course_authors/source/getting_started/get_started.rst]: The language in the rest of the file is very edx-specific, so I assume this disclaimer is just for those who happen on this documentation in accident?

File: en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst [hunk 9]
```diff
@@ -1,134 +1,151 @@
...
-* Draft a Rough Course Outline
-* Explore edX's Support Tools
-* Draft Your Course About Page
+.. * Getting Started with Studio
+.. * Draft a Rough Course Outline
+.. * Explore edX's Support Tools
+.. * Draft Your Course About Page
 
-From the **Tools** menu, select **Checklists**.
+.. From the **Tools** menu, select **Checklists**.
```
Comment (sarina) [inline en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst]: So... the checklist is wrong, but still present in the codebase? Perhaps we should remove it from there too?

File: en_us/shared/building_and_running_chapters/building_course/scheduling_course.rst [hunk 10]
```diff
@@ -171,30 +154,73 @@ Advertise a Different Start Date
...
+
+#. Locate the **Course Advertised Start Date** field. The default value is
+   ``null``.
+
+#. Enter the start date that you want learners to see for your course. Date
+   values entered in MM/DD/YYYY format will appear to learners in DD Mon YYYY
+   format.
+
+#. Add quotation marks (``" "``) before and after the start date value. An
+   example follows.
```
Comment (sarina) [inline en_us/shared/building_and_running_chapters/building_course/scheduling_course.rst]: It sounds like you need to follow both this step and the previous one, but I think what you're saying is that you can enter in _either_ a MM/DD/YYYY formatted date, or any random string - is that right?

File: en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst [hunk 11]
```diff
@@ -256,8 +283,8 @@ Replace the placeholders in the following template with your information.
       <h2>Frequently Asked Questions</h2>
       <article class="response">
         <h3>Do I need to buy a textbook?</h3>
-        <p>No, a free online version of Chemistry: Principles, Patterns, and 
-        Applications, First Edition by Bruce Averill and Patricia Eldredge 
+        <p>No, a free online version of Chemistry: Principles, Patterns, and
+        Applications, First Edition by Bruce Averill and Patricia Eldredge
```
Comment (sarina) [inline en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst]: Can you check what the default questions are? We removed this question in favor of another one (Open edX users were getting confused wondering where the free textbook was)

**Comment by sarina** [comment]
LGTM.

File: en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst (hunk 9 above)
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst]: Correct, they are wrong in the codebase, @sarina . One of my tasks is to follow up with @explorerleslie to see if there is any way we could find a way to complete her hackathon project from some months back that brought the checklists up to date. Her work got hung up on failing tests.

Comment (sarina) [inline en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst]: Do you know where that pull request might be? failing tests are easy to fix.

File: en_us/shared/building_and_running_chapters/building_course/scheduling_course.rst (hunk 10 above)
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/scheduling_course.rst]: yes, bothe of the steps, adding the value and then enclosing it in quotes. @sarina would this be clearer if the "Date
 values entered in MM/DD/YYYY format will appear to learners in DD Mon YYYY format." sentence was located in a different spot? maybe describe that you can enter either a string or a date in a specific format, which will then be redisplayed in a different format... best up around lines 157-9?

File: en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst (hunk 11 above)
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst]: sure @sarina -- can you give me a hint where I would check that?

File: en_us/course_authors/source/getting_started/get_started.rst (hunk 8 above)
Comment (lamagnifica) [inline en_us/course_authors/source/getting_started/get_started.rst]: ugh, that's awful. This file only appears in the partner B&R guide, @sarina , not in the open edX one. I'll reword and redirect people to the other guide for more relevant info.

File: en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst (hunk 11 above)
Comment (sarina) [inline en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst]: Probably in any devstack about page?

File: en_us/shared/building_and_running_chapters/building_course/scheduling_course.rst (hunk 10 above)
Comment (sarina) [inline en_us/shared/building_and_running_chapters/building_course/scheduling_course.rst]: maybe as simple as doing:

Locate the **Course Advertised Start Date** field. The default value is `null`. To change this value, you can either:
- Enter the start date that you want learners to see for your course. Date values entered in MM/DD/YYYY format will appear to learners in DD Mon YYYY format. OR,
- Add quotation marks (`" "`) before and after the start date value. An example follows.

Comment (sarina) [inline en_us/shared/building_and_running_chapters/building_course/scheduling_course.rst]: or put the "OR" at the beginning of the "add quotation marks" line.

File: en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst (hunk 9 above)
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst]: https://openedx.atlassian.net/browse/PR-617 is the story, now closed. Also mentioned on the Hacakathon 9 page with a very stale sandbox link only. I could not find a PR. I mentioned this to Leslie, and she would love to get it finished up, but can't get the time until after Global Forum. 
https://openedx.atlassian.net/wiki/display/OPEN/Hackathon+Nine%3A+Hacking+is+%3C3

File: en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst (hunk 11 above)
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst]: hmm, @sarina running studio in Devstack fails for me. I'll ask my colleagues in doc if one of them can check. I did check Edge, though, and this info remains unchanged on Edge.

Comment (sarina) [inline en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst]: Ah, I changed it in the demo course, but not the new course templates. I'll
do that, and tag you on the pr.

Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst]: Sweet!

File: en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst [hunk 12]
```diff
@@ -86,49 +105,47 @@ team privileges.
   :width: 600
   :alt: Image of the course on the Studio dashboard
 
-To open a course, select the course name. The Studio **Course Outline** page
+To edit a course, select the course name. The Studio **Course Outline** page
 appears.
 
-.. _Use the Course Checklist:
+.. Topic commented out per M. Rudnick as the course checklist is sending people to info that is not just out of date, it's bad. - Alison 30 Oct 15
```
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst]: rethink this -- there is a context-sensitive link to this section from the help!

Comment (srpearce) [inline en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst]: Can we get rid of that link?

File: en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst [hunk 13]
```diff
@@ -138,19 +155,28 @@ prerequisites.
...
 
-To add a course team member:
+=========================
+Add a Course Team Member
+=========================
+
+.. important:: Course team members are not automatically enrolled in courses,
+   although they can access content in Studio, the LMS, and Insights. To work
+   on a course, every course team member must register, activate a user
+   account, and enroll in the course.
```
Comment (srpearce) [inline en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst]: When I saw "register" I automatically thought it meant "enroll in the course" (even though I know that we use "register" and "enroll" differently!). To make this just a little clearer, I'd say something like "every course team member must create and activate an edX account, and then enroll in the course."

File: en_us/shared/building_and_running_chapters/building_course/scheduling_course.rst [hunk 14]
```diff
@@ -171,30 +154,73 @@ Advertise a Different Start Date
 ====================================
 
 You can advertise a start date for your course that is different from the
-course start date you set in the **Schedule and Details** page. You might want
+course start date you set in the **Schedule & Details** page. You might want
 to do this if there is uncertainty about the exact start date. For example, you
-could advertise the start date as **Spring, 2015**.
+could advertise the start date as Spring, 2016.
```
Comment (srpearce) [inline en_us/shared/building_and_running_chapters/building_course/scheduling_course.rst]: Would change to something non-hemisphere-specific, such as "July" or "mid-2016".

File: en_us/shared/building_and_running_chapters/building_course/scheduling_course.rst [hunk 15]
```diff
@@ -171,30 +154,73 @@ Advertise a Different Start Date
...
+and, after they enroll in the course, on their dashboard.
+
+.. image:: ../../../shared/building_and_running_chapters/Images/about-page-course-start.png
+ :alt: The course About page, showing the start date.
+ :width: 800
+
+.. only:: Open_edX
+
+  To find the URL of your course's **About** page, in Studio select
+  **Settings** and then **Schedule & Details**.
```
Comment (srpearce) [inline en_us/shared/building_and_running_chapters/building_course/scheduling_course.rst]: This reads just a little awkwardly. Maybe "To find the URL of your course's About page, go to Studio, select **Settings**, and then select **Schedule & Details**"?

File: en_us/shared/building_and_running_chapters/building_course/scheduling_course.rst [hunk 16]
```diff
@@ -171,30 +154,73 @@ Advertise a Different Start Date
...
+
+After learners enroll in your course, the course appears on their course
+dashboards. To access the dashboard, learners select their usernames and then
+**Dashboard**.
+
+.. image:: ../../../shared/building_and_running_chapters/Images/dashboard-course-start-and-end.png
+ :alt: The student dashboard with a course in progress, one that has ended, one
```
Comment (srpearce) [inline en_us/shared/building_and_running_chapters/building_course/scheduling_course.rst]: student --> learner

File: en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst [hunk 17]
```diff
@@ -46,39 +48,50 @@ You configure the contents of this page in Studio, as described in this topic.
...
+ :alt: A course About page with the description circled.
  :width: 600
 
-.. note:: For courses on edX.org, you must communicate the course description
- to your edX Partner Manager, to ensure the content is accurate on the course
- About page.
+.. only:: Partners
+
+  .. note:: For courses running on edX.org, you must communicate the course
+   description to your edX Partner Manager, to ensure the content is accurate
```
Comment (srpearce) [inline en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst]: Strike comma; decap "partner manager"

File: en_us/shared/building_and_running_chapters/building_course/course_rerun.rst [hunk 18]
```diff
@@ -75,7 +75,7 @@ Using Re-Run to Create a Course
 
 .. only:: Partners
 
-   Only edX Partner Managers have the permissions needed to re-run a course.
+   Only edX staff members have the permissions needed to re-run a course.
    To re-run a course that is hosted on the `edx.org`_ or `edge.edx.org`_
    sites, contact your edX Partner Manager. After your Partner Manager creates
```
Comment (srpearce) [inline en_us/shared/building_and_running_chapters/building_course/course_rerun.rst]: Decap partner manager (unless we all want to capitalize our job titles!) here and elsewhere

File: en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst [hunk 19]
```diff
@@ -101,10 +115,11 @@ page.
 Add a Course Image
 ************************
 
-The course image that you add in Studio appears on the dashboard. It should be
-a minimum of 378 pixels in width by 225 pixels in height, and in .jpg or .png
-format. Make sure the image that you upload maintains the aspect ratio of those
-dimensions so that the image appears correctly on the dashboard.
+The course image is an image file that you upload to Studio appears on the
```
Comment (srpearce) [inline en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst]: _and that_ appears

File: en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst [hunk 20]
```diff
@@ -101,10 +115,11 @@ page.
...
+learner dashboard. The image file should be a minimum of 378 pixels in width by
```
Comment (srpearce) [inline en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst]: should or must?

File: en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst [hunk 21]
```diff
@@ -162,29 +172,43 @@ time of less than 2 minutes.
 Ensure your course introduction video follows the same :ref:`Compression
 Specifications` and :ref:`Video Formats` guidelines as course content videos.
 
+Learners access the course video from the course **About** page. In this
+example, the video **play** option appears over the course image. When learners select **play**, the video player opens and the course video plays.
```
Comment (srpearce) [inline en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst]: line wrap

**Comment by srpearce** [comment]
A few minor comments; otherwise :+1:

File: en_us/shared/building_and_running_chapters/building_course/course_rerun.rst (hunk 18 above)
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/course_rerun.rst]: I searched the doc files, and there are 38 instances where we capitalize Partner Manager, and only one (in release notes) where we lc this term. How to jump? @catong @srpearce

File: en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst (hunk 12 above)
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/creating_new_course.rst]: well, @srpearce , the notion was that we would temporarily hide this section in doc until we could resurrect Leslie's hackathon project to make the checklists useful and not actually detrimental (which was derailed only by failing tests), tentatively "after GF".

File: en_us/shared/building_and_running_chapters/building_course/scheduling_course.rst (hunk 10 above)
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/scheduling_course.rst]: I believe that you always need the quotation marks.

Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/scheduling_course.rst]: ^ you always need the quotation marks

File: en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst (hunk 11 above)
Comment (lamagnifica) [inline en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst]: @sarina didn't want to lose track of this. Do you have a story for changing the default questions/reference to textbooks?

Comment (sarina) [inline en_us/shared/building_and_running_chapters/building_course/setting_up_student_view.rst]: Yes, thanks for the reminder. https://github.com/edx/edx-platform/pull/10678

**Comment by lamagnifica** [comment]
Closing in favor of PR#723. https://github.com/edx/edx-documentation/pull/723

**Pull request closed without merging**